### PR TITLE
Render StatsBoxGrid as a span [Fixes #4757]

### LINKED ISSUE #4757: StatsBoxGrid result rendering as an H3
**Describe the bug**

The StatsBox with charts on our homepage currently uses the H3 tag to render the result. I'm guessing this was done as a quick way to get font styling but it is poor for accessibility and SEO. 

Screenshot 2021-12-13 at 12 03 21

**To Reproduce**

1. Go to 'https://ethereum.org/'
2. Scroll down to 'The latest network statistics'
3. See error

**Expected behavior**

Result should be rendered in a span that is explicitly styled to look like an H3.

**Want to contribute?**

We love contributions from the Ethereum community! Please comment on an issue if you're interested in helping out with a PR.

### PR DESCRIPTION
**Pull request by joneskj55**
## Description
This PR renders the StatsBoxGrid result as a `span` as opposed to an `h3` element.

## Related Issue
#4757

## With change
Screen Shot 2021-12-14 at 12 48 06 PM

### CODE REVIEW

**Review by wackerow — APPROVED** [review]
@joneskj55 Thanks! LGTM =)

**Review by corwintines — APPROVED** [review]
Thanks @joneskj55!

File: src/components/StatsBoxGrid.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ import Icon from "./Icon"
 import { isLangRightToLeft } from "../utils/translations"
 import { getData } from "../utils/cache"
 
-const Value = styled.h3`
+const Value = styled.span`
   position: absolute;
   bottom: 8%;
   font-size: min(4.4vw, 64px);
```
Comment (minimalsm) [inline src/components/StatsBoxGrid.js:18]:
```suggestion
  font-size: min(4.4vw, 4rem);
```

**Review by minimalsm — APPROVED** [review]
Thanks for this @joneskj55.

Made a small tweak from px to rem in the [name of accessibility](https://www.24a11y.com/2019/pixels-vs-relative-units-in-css-why-its-still-a-big-deal/?ref=heydesigner) while I was here :-)

**Comment by joneskj55** [comment]
@minimalsm Cool, and thanks for the a11y link!

**Pull request merged**